sparkbox/code-challenge-gilded-rose
# Add Jest Config

## Description

This PR adds support for Jest to be used as a test runner in the Gilded Rose challenge. In addition, it adds support for Babel in anticipation of an ES6 refactor.

#### Validation Steps
1. Pull down `test--add-jest`.
1. Run `npm i`.
1. Run `npm test`.
1. - [x] Confirm all tests pass.
